megatron/optimizer/optimizer.py

```diff
@@ -664,13 +664,6 @@ def _copy_main_params_to_model_params(self):
                                         overflow_buf=self._dummy_overflow_buf)
 
 
-    def _copy_model_params_to_main_params(self):
```

Comment:
Hello, this function is different from the `_copy_main_params_to_model_params`. See the params to `_multi_tensor_copy_this_to_that`. For `_copy_model_params_to_main_params`, model_data->main_data whereas for `_copy_main_params_to_model_params`, it is main_data->model_data.